src/ogdc_runner/__main__.py

```diff
@@ -59,7 +50,7 @@ def check_workflow_status(workflow_name: str) -> None:
 
 @cli.command
 @recipe_path
-def submit_and_wait(recipe_path: Path) -> None:
+def submit_and_wait(recipe_path: str) -> None:
```

Comment:
We don't need to change this now, but I'm wondering if maybe the CLI entrypoint should be responsible for reading the recipe from the provided path, and then pass that along to relevant functions instead of the recipe path. That way we can catch exceptions related to e.g., misformed path string early on, and other parts of the code don't need to leverage fsspec directly.